AlfrescoKit/AlfrescoKit/Supporting Files/Localizable.strings

```diff
@@ -107,4 +107,4 @@
 "ak.naming.view.controller.title" = "Name";
 "ak.naming.view.controller.cell.title" = "Name";
 "ak.naming.view.controller.cell.name.placeholder" = "Name here";
-"ak.naming.view.controller.upload.button.title" = "Upload";
+"ak.naming.view.controller.upload.button.title" = "Select";
```

Comment:
Might be worth renaming the localisation key if the action is no longer always "Upload"